sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -240,8 +240,6 @@ public class ConfigurationLiveTests: RecordedTestBase<AppConfigurationTestEnviro
 
 When tests are run in recording mode, session records are saved to the project directory automatically in a folder named 'SessionRecords'.
 
-__NOTE:__ recordings are copied from `netcoreapp2.1` directory by default, make sure you are running the right target framework.
```

Comment:
Should this have a corresponding comment for netcoreapp3.1?

Reply:
We removed the need to copy recordings so now any framework you run in recording mode would update recordings.